ext/winevt/winevt_subscribe.c

```diff
@@ -1,5 +1,29 @@
 #include <winevt_c.h>
 
+/*
+ * Document-class: Winevt::EventLog::Subscribe
+ *
+ * Subscribe Windows EventLog channel.
+ *
+ * @example
+ *  require 'winevt'
+ *
+ *  @subscribe = Winevt::EventLog::Subscribe.new
+ *  @subscribe.tail = true
+ *  @subscribe.rate_limit = 80
+ *  @subscribe.subscribe(
+ *    "Application", "*[System[(Level <= 4) and TimeCreated[timediff(@SystemTime) <= 86400000]]]"
+ *  )
+ *  while true do
+ *    @subscribe.each do |eventlog, message, string_inserts|
+ *      puts ({eventlog: eventlog, data: message})
+ *    end
+ *    sleep(0.1)
+ *  end
+ *
+ * @see: https://docs.microsoft.com/en-us/windows/win32/api/winevt/nf-winevt-evtsubscribe
```

Comment:
The trailing `:` for `@see` seems break the output document.

Reply:
Fixed in https://github.com/fluent-plugins-nursery/winevt_c/pull/13/commits/d1480632018c5e684e8e6dcc53faa855ec810ba7.